api/types/types.go

```diff
@@ -58,16 +64,18 @@ type ImageHistory struct {
 	Comment   string
 }
 
+// ImageDelete contains response of Remote API:
 // DELETE "/images/{name:.*}"
 type ImageDelete struct {
 	Untagged string `json:",omitempty"`
 	Deleted  string `json:",omitempty"`
 }
 
+// Image contains response of Remote API:
 // GET "/images/json"
 type Image struct {
 	ID          string `json:"Id"`
-	ParentId    string
+	ParentID    string
```

Comment:
As it is *serialized* as JSON, I think it would be better to do something like :

```go
ParentID string `json:ParentId`
```

This way, this limit the possible *breakage* of tools using the API.